openpype/hosts/maya/api/lib_renderproducts.py

```diff
@@ -1059,7 +1074,7 @@ def get_render_products(self):
                 continue
 
             aov_type = self._get_attr(aov, "aovType")
-            if multipart and aov_type not in self.unmerged_aovs:
+            if self.multipart and aov_type not in self.unmerged_aovs:
```

Comment:
```suggestion
            multipart = self.get_multipart()
            if multipart  and aov_type not in self.unmerged_aovs:
```
If I dont get it wrong...

Reply:
> If I dont get it wrong...

I think:

```python
multipart = self.get_multipart()
```
is already done in the constructor of the abstract base class.

Reply:
i see. i will resume my statement.

Reply:
> @moonyuet sorry I'm not sure what this does?

just leaves this comment alone, I didn't notice there is the constructor of the base class done for the `multipart.`